.github/workflows/main.yml

```diff
@@ -87,7 +87,7 @@ jobs:
         if: startsWith(matrix.package, 'examples/')
       - name: Install package
         run: pip install ./$(echo "${{ matrix.package }}[dev]" | sed 's#/revolve2##')
-        if: "!startsWith(matrix.package, 'examples/')"
+        if: "(!startsWith(matrix.package, 'examples/') && !startsWith(matrix.package, 'tests'))"
```

Comment:
are the outer brackets needed?

Reply:
yes without it didnt work for some reason

Reply:
🤷